package.json

```diff
@@ -260,6 +260,7 @@
     "jest-html-reporter": "^3.5.0",
     "jest-junit": "^10.0.0",
     "jest-snapshot": "^26.1.0",
+    "lokijs": "^1.5.12",
```

Comment:
this is needed to run a wallet connect client on the server side. i put it in dev dependencies since we're just using it for the tests